animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -88,25 +95,40 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
             col.prop(self, "distanceMode", text = "")
         if self.mode == "MESH":
             col.prop(self, "meshMode", text = "")
+        if self.mode == "GRID":
+            row = col.row(align = True)
+            row.prop(self, "centerAlongX", text = "X", toggle = True)
+            row.prop(self, "centerAlongY", text = "Y", toggle = True)
+            row.prop(self, "centerAlongZ", text = "Z", toggle = True)
 
     def drawAdvanced(self, layout):
         if self.mode == "CIRCLE":
             layout.prop(self, "exactCircleSegment")
 
-    def getExecutionFunctionName(self):
+    def getExecutionCode(self, required):
         if self.mode == "LINEAR":
-            return "execute_Linear"
+            if self.distanceMode == "STEP":
+                list = ["matrices = self.execute_Linear(amount, distance)"]
```

Comment:
Don't use `list` as a variable name.
Also `yield` can be used to instead of explicitely creating a list.